### PR TITLE
mongo-orchestration 0.5 (new formula)

### PR DESCRIPTION
**Pull request by alcaeus**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by mikemcquaid** [comment]
test doesn't work, looks like it needs to be given some arguments to do something: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/42299/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_mongo_orchestration/

**Comment by alcaeus** [comment]
Yeah, I'll take a look at it - I thought it would be enough if the tool left output - apparently not ;)

File: Library/Formula/mongo-orchestration.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    sha256 "57a86ca602b0a4d2da1f9f3afa8c59fd8ca62d829f6d8f467eae0b7cb22ba88a"
+  end
+
+  resource "cherrypy" do
+    url "https://pypi.python.org/packages/source/C/CherryPy/CherryPy-4.0.0.tar.gz"
+    sha256 "73ad4f8870b5a3e9988a7778b5d3003a390d440527ec3458a0c7e58865d2611a"
+  end
+
+  def install
+    vendor_site_packages = libexec/"vendor/lib/python2.7/site-packages"
```
Comment (DomT4) [inline Library/Formula/mongo-orchestration.rb]: If we're not using this elsewhere just inline it below.

**Comment by alcaeus** [comment]
@mikemcquaid Test fixed, also fixed a minor issue with the plist file. It's good from my end, let me know if there's anything else that needs changing.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**